glide.lock

```diff
@@ -0,0 +1,52 @@
+hash: d576153bffcb3088cc901b219fcac5ff48dfe76de64f8eec66c7ad730ae6a3aa
```

Comment:
It still makes sense to track dependency versions here using Glide despite them being vendored, because with Glide we can upgrade them in a more systematic way.